packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -628,7 +635,7 @@ export function usePress(props: PressHookProps): PressResult {
           disableTextSelection(state.target);
         }
 
-        let shouldStopPropagation = triggerPressStart(e, state.pointerType);
+        let shouldStopPropagation = triggerPressStart(initEventCoordinates(e, state.target), state.pointerType);
```

Comment:
this is a touch event, it should already have a clientX and clientY if you go through the TouchList https://developer.mozilla.org/en-US/docs/Web/API/Touch apologies for saying a bunch of these were missed, I hadn't quite thought through it all the way, I think it was only a couple

the other touch events should also have those properties

However, maybe we don't worry about the press events which come from touchstart/touchend and mousedown/mouseup, the pointer events are supported on all major browsers, so theoretically these are never being hit except in jsdom tests where pointer events are not supported yet

I'll see how the rest of the team feels, but for now, don't worry about them.

you've found an interesting set, what to do for canceled press. it can happen for a few reasons, one of which is that the cursor is no longer on the target. i think you're doing the right thing by not treating it differently